packages/components/src/components/hds/stepper/step/indicator.ts

```diff
@@ -6,18 +6,30 @@
 import Component from '@glimmer/component';
 import { assert } from '@ember/debug';
 
-export const DEFAULT_STATUS = 'incomplete';
-export const STATUSES = ['incomplete', 'progress', 'processing', 'complete'];
+import { HdsStepperStatusesValues } from '../types.ts';
+import type { HdsStepperStatuses } from '../types.ts';
 
-export default class HdsStepperIndicatorStepIndexComponent extends Component {
+export const DEFAULT_STATUS = HdsStepperStatusesValues.Incomplete;
+export const STATUSES: string[] = Object.values(HdsStepperStatusesValues);
+
+interface HdsStepperIndicatorStepSignature {
+  Args: {
+    isInteractive?: boolean;
+    status: HdsStepperStatuses;
```

Comment:
See: https://helios.hashicorp.design/components/stepper?tab=code#stepperstepindicator
```suggestion
    status: HdsStepperStatuses;
    isInteractive?: boolean;
```